### ISSUE TITLE
Sudden stop with extreme high acceleration

### ISSUE THREAD

**Issue by gin66** [gin66/FastAccelStepper#46]
This command causes a complete stop at high speed:
```
M1 V40 a10000 A1000000000 w5000 V50 U w100 V40 U
```

**Comment by gin66** [comment]
This works:
```
M1 V40 a10000 A60000000 w5000 V50 U w100 V40 U
```
This fails:
```
M1 V40 a10000 A70000000 w5000 V50 U w100 V40 U
```

**Comment by gin66** [comment]
Fixed in 0.20.2
